### DISCUSSION TITLE
Next.js 14 Production Issue: TypeError on 'READY' property - Need Assistance

### DISCUSSION THREAD

**Discussion by shadwan** [vercel/next.js#61144] [Help]
### Summary

Hello Next.js and JavaScript communities,

I am currently facing a perplexing issue with my Next.js application (version 14.1.0 with approuter) that's only occurring in production mode. The application runs smoothly in a development environment, but the production deployment is encountering a TypeError that I'm struggling to resolve.

**Error Description:**

```

TypeError: Cannot read properties of undefined (reading 'READY')
    at i (/home/app/.next/server/chunks/4221.js:1:37706)
    ...

```

This error seems to stem from a chunk file in the Next.js build, but the exact cause is elusive.

Environment Details:

Operating System: Linux (Ubuntu), platform: linux, arch: x64
Node Version: 18.19.0
NPM Version: 10.2.3
Yarn Version: 1.22.19
Next.js Version: 14.1.0
React/React-DOM: 18.2.0
Deployment: Digital Ocean Droplet inside Docker
Other Notes: The issue is not reproducible in a local production build or in the development environment.
I've checked for compatibility issues and have ensured that all dependencies are up to date. Given that the error does not occur in development, I suspect it might be related to the production build process or a specific environment configuration within Docker.

If anyone has encountered a similar issue or has insights into potential causes, your advice would be greatly appreciated. I'm particularly interested in any Docker-specific configurations or limitations that might affect Next.js applications, or any known issues with the mentioned versions of Node, Next.js, or React.

Thank you in advance for your time and help!

### Additional information

```js
Operating System:
  Platform: linux
  Arch: x64
  Version: #9-Ubuntu SMP PREEMPT_DYNAMIC Sat Oct  7 01:35:40 UTC 2023
Binaries:
  Node: 18.19.0
  npm: 10.2.3
  Yarn: 1.22.19
  pnpm: N/A
Relevant Packages:
  next: 14.1.0
  eslint-config-next: N/A
  react: 18.2.0
  react-dom: 18.2.0
  typescript: N/A
Next.js Config:
  output: N/A
```

**Comment by sam-hieken** [comment]
Hey OP, I had a similar issue on Next 14: I received `TypeError: t is undefined` on my production server, despite no issues occurring in my locally hosted production server, nor my development server. I ended up deleting the `.next` folder and rebuilding the production server which solved the issue.

Maybe a weird bug like that occurred for you too? Deleting it might be worth a try. Keep in mind I did this after re-`npm run build`ing the production server multiple times, clearing cache, and switching browsers, so don't rule it out if you've already tried those.
